src/ray/object_manager/plasma/BUILD.bazel

```diff
@@ -0,0 +1,399 @@
+        "//:object_manager_common",
+        "//src/ray/common:status",
+        "//src/ray/protobuf:common_cc_proto",
+        "@com_google_absl//absl/container:flat_hash_map",
+    ],
+)
+
+ray_cc_library(
+    name = "plasma_allocator",
+    srcs = ["plasma_allocator.cc"],
+    hdrs = ["plasma_allocator.h"],
+    deps = [
+        ":plasma_allocator_interface",
+        ":plasma_malloc",
+        "//:object_manager_common",
+        "//src/ray/common:ray_config",
+        "//src/ray/util:logging",
+        "@com_google_absl//absl/types:optional",
+    ],
+)
+
+ray_cc_library(
+    name = "plasma_create_request_queue",
+    srcs = ["create_request_queue.cc"],
+    hdrs = ["create_request_queue.h"],
+    deps = [
+        ":object_manager_plasma_common",
+        ":plasma_connection_protocol",
+        "//:object_manager_common",
+        "//src/ray/common:file_system_monitor",
+        "//src/ray/common:status",
+        "@com_google_absl//absl/container:flat_hash_map",
+    ],
+)
+
+ray_cc_library(
+    name = "object_manager_plasma_common",
+    srcs = ["plasma.cc"],
+    hdrs = [
+        "common.h",
+        "plasma.h",
+    ],
+    deps = [
+        ":plasma_generated",
+        "//:object_manager_common",
+        "//src/ray/common:id",
+        "//src/ray/util:compat",
+        "//src/ray/util:macros",
+        "@boost//:asio",
+        "@com_google_googletest//:gtest_prod",
+    ],
+)
+
+ray_cc_library(
+    name = "plasma_generated",
+    hdrs = ["plasma_generated.h"],
+    deps = ["@com_github_google_flatbuffers//:flatbuffers"],
+)
+
+ray_cc_library(
+    name = "plasma_connection_protocol",
+    srcs = [
+        "connection.cc",
+        "protocol.cc",
+    ],
+    hdrs = [
+        "connection.h",
+        "protocol.h",
+    ],
+    deps = [
+        ":fling",
+        ":object_manager_plasma_common",
+        ":plasma_generated",
+        "//:object_manager_common",
+        "//src/ray/common:id",
+        "//src/ray/common:network",
+        "//src/ray/common:status",
+        "//src/ray/protobuf:common_cc_proto",
+        "//src/ray/util:compat",
+        "//src/ray/util:logging",
+        "@com_github_google_flatbuffers//:flatbuffers",
+        "@com_google_absl//absl/container:flat_hash_map",
+        "@com_google_absl//absl/container:flat_hash_set",
+    ],
+)
+
+ray_cc_library(
+    name = "fling",
+    srcs = select({
+        "@platforms//os:windows": [
+        ],
+        "//conditions:default": [
+            "fling.cc",
+        ],
+    }),
+    hdrs = select({
+        "@platforms//os:windows": [
+        ],
+        "//conditions:default": [
+            "fling.h",
+        ],
+    }),
+    deps = select({
+        "@platforms//os:windows": [
+        ],
+        "//conditions:default": [
+            "//src/ray/util:logging",
+        ],
+    }),
+)
+
+flatbuffer_cc_library(
+    name = "plasma_fbs",
+    srcs = ["plasma.fbs"],
+    flatc_args = FLATC_ARGS,
+    out_prefix = "",
+)
+
+ray_cc_test(
```

Comment:
Just FYI: I don't have a strong preference here, but generally BUILD file lives under the same directory as the source file (which is also the purpose for this issue :) )

Reply:
Sure! I checked the `/src/ray/common` and it put the tests at seperate BUILD file.